content/events/fosdem/index.adoc

```diff
@@ -0,0 +1,48 @@
+---
+layout: project
+title: "Jenkins at FOSDEM"
+sigId: "advocacy-and-outreach"
+tags:
+  - outreach-programs
+  - community
+  - events
+  - fosdem
+links:
+  gitter: jenkinsci/fosdem
+---
+
+link:https://fosdem.org/[FOSDEM] is a regular event for open-source projects developers and users.
+It happens very year in Brussels (late January or early February).
+Jenkins project regularly participates in this event and organizes additional events around the FOSDEM dates.
+
+=== FOSDEM 2020
+
+Jenkins project will participate in FOSDEM.
+This event is coordinated by the link:/sigs/advocacy-and-outreach/[Jenkins Advocacy and Outreach SIG].
+Current plan:
+
+* **Jan 30** - Jenkins Pipeline Fundamentals training led by link:/blog/authors/MarkEWaite/[Mark Waite] (link:https://www.eventbrite.com/e/jenkins-pipeline-fundamentals-training-tickets-87080214265[Register here])
+* **Jan 31** - Jenkins Contributor Summit (link:https://www.meetup.com/jenkinsmeetup/events/267684785/[RSVP here])
```

Comment:
I sent a reservation request to Le Roy d'Espagne (for 20 people), and I am waiting for a confirmation